uiowa/uiowa
# Quote block type

Closes #2881 
# How to test

```
rm -rf ./node_modules
yarn cache clean
yarn install
yarn workspace uids_base gulp --development
blt ds --site=sandbox.uiowa.edu
drush @sandbox.local uli
```
- Test on v2 to see if it still works, the only change should be that the cite url displays below the footer instead of inline. 

<img width="234" alt="Screen Shot 2021-11-12 at 12 12 34 PM" src="https://user-images.githubusercontent.com/1036433/141514871-a00a63ae-ab7d-4d37-927a-01ea40cf69b0.png">

- Test on v3

## Still Todo
- [x] Will need to merge blockquote [PR](https://github.com/uiowa/uids/pull/603) in uids so can commit to hash - so PLEASE DON'T SOLVE MERGE CONFLICT!
